keps/sig-node/2400-node-swap/README.md

```diff
@@ -727,15 +916,21 @@ Here are specific improvements to be made:
 - Add e2e test confirming that swap is used for `LimitedSwap`.
 - Document [best practices](#best-practices) for setting up Kubernetes with swap
 
-[via cgroups]: #restrict-swap-usage-at-the-cgroup-level
+#### Beta 3
 
-#### GA
+- Enhance website documentation
+  - Docs should be close to GA quality before promoting to GA.
+- Make eviction manager swap aware.
 
-_(Tentative.)_
+#### GA
 
-- Test a wide variety of scenarios that may be affected by swap support.
-- Remove feature flag.
-- Remove the Swap Support using Burstable QoS Pods only deprecated in Beta 2.
+- Test a wide variety of scenarios that may be affected by swap support, including tests with aggressive memory stress.
+- Address memory-backed backed volumes which should not have access to swap.  
+- Remove feature gate.
+- Exclude high-priority, static and mirrored pods from gaining access to swap.
```

Comment:
Should we move this to Beta 3 to avoid unintentional regression (i.e., such pods can gain access to swap in Beta 3 but not in GA)?

Reply:
Definitely, thank you!
Moved most of these to beta3 as they're already fulfilled.